src/js/common/schemaform/actions.js

```diff
@@ -81,15 +82,22 @@ export function submitForm(formConfig, form) {
         return res.json();
       }
 
-      return Promise.reject(new Error(res.statusText));
+      return Promise.reject(new Error(`vets_server_error: ${res.statusText}`));
```

Comment:
I'm pretty confident, but still want to ask: This `then()` will only be reached if we actually get a _response_ from the server, right? If there's a network issue, it'll bypass here and go straight to the following `catch()`, right? I assume you tested it by throttling your network connection to nil? (If not, I can always give it a go for my own peace of mind.)